<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <style>
        body {
            font-family: "Century Gothic", sans-serif;
        }
    </style>
    <style>
        .container {
            display: flex; /* Define um contêiner flexível */
            align-items: center; /* Alinha itens verticalmente ao centro */
        }
        .image {
            width: 30%; /* Define a largura máxima da imagem */
            height: auto;
            margin-right: 20px; /* Margem embaixo para separar a imagem do parágrafo */
            margin-bottom: 0px;
            max-width: 100%;
        }
    </style>
</head>

<body>
  <div style="text-align:center; font-size: 20px; font-weight: bold;"> 
    Hello, World!
  </div>
</body>
<p style='text-align: justify;'>
<body>
    <div class="container">
        <img src="assets/images/eu2.jpg" alt="Imagem" class="image">
        <p style='text-align: justify;'>I'm Thiago Mendonça a researcher currently in the final stages of completing my Ph.D. in Bioinformatics. My initial exposure to bioinformatics occurred during my undergraduate exchange program in Canada. The very first lesson was all about the Monte-Carlo Algorithm, followed by 5 hours of R practice. It was a bit of a shock to me, so much math and computation! I was determined to avoid bioinformatics as much as possible. Until my Master's program, I had been exclusively involved in wet lab molecular biology techniques. </p>
    </div>
</body>
<body><p style='text-align: justify;'>
    However, my project in the Master's program required me to dig into transcriptomics, which meant 100% in silico work. To my surprise, I found myself developing an affinity for it. The endless possibilities and beautiful graphs were joy to my eyes. Now, I spend my time conducting detailed and in-depth computational analyses using programming languages such as Python, R, and Bash to perform various biological experiments and analyses for <strong>research</strong> and <strong>private companies</strong>. My areas of interest include <strong>Genomics</strong> and <strong>Structural Biology</strong> (It's very exciting when I combine both), specially for <strong>Virology</strong>. Take a quick look at some examples!</p>
</body>
<br>
<body>
  <div style="text-align:center;">
    <h1 style="color:green;">Presenting Data in Graphs</h1>
  </div>
</body>
<hr style="width:100%;text-align:center;">
<body>
  <div style="text-align:center; font-size: 20px; font-family:century gothic; font-weight: bold;"> 
    Project I
  </div>
</body>
<p align="center">
 "Pangenome of putative viral family"
 </p>
<p align="center">
  <img src="assets/images/1_2.svg" style="max-width: 100%; height: auto;" alt="Collection of graphics">
</p> 
<p>1- FlowerPlot - intersections of gene content</p> 
<p>2- Pangenomic rarefraction curve</p> 
<p>3- Normalized heatmap of gene intersections</p> 
<p>4- Heaps law calculation</p> 
<p>5- PCA plot of gene content</p> 
<p>6- Boxplot of genomic features</p> 
<br>
<br>
<body>
  <div style="text-align:center; font-size: 20px; font-weight: bold;"> 
    Project II
  </div>
</body>
<p align="center">
 "Genomic and Structural Analysis of Major Vault Potein in Bacteriophages"
 </p>
<p align="center">
  <img width="1160" height="1000" src="assets/images/2_2.svg">
</p>
<p>7- Genomic representation of bacteriophages' genome, Reads mapping to assembled genome, Phylogenetic tree of new bacteriophages and reference ones</p>
<p>8- Phylogenetic tree of Eukaryotic, Prokaryotic and viral MVPs</p>
<p>9- Tertiary tructures of different MVPs compared do modelled phage MVPs</p>
<br>
<br>
<body>
  <div style="text-align:center; font-size: 20px; font-weight: bold;"> 
    Project III
  </div>
</body>
<p align="center">
 "Characterization of SARS-CoV-2 and Human mutations on protein binding affinity"
</p>
<p align="center">
  <img width="1360" height="1200" src="assets/images/3_2.svg">
</p>
<p>10- Close-up of docked Spike and ACE2 protein, showing the interacting mutated aminoacids.</p>
<p>11- Heatmap of formed Hydrogen Bonds.</p>
<p>12- Boxplots of binding affinities between different ACE2 and Spike proteins.</p>
<p>13- RMSD calculated from Molecular Dynamics of different ACE2/Spike complexes.</p>
<br>
<br>
<body>
  <div style="text-align:center; font-size: 20px; font-weight: bold;"> 
    Miscellaneous
  </div>
</body>
<p align="center">
  <img width="1360" height="1200" src="assets/images/4_2.svg">
</p>
<p>14- Protein-Protein Network generated from STRING and visualized in Cytoscape and Gephi.</p>
<p>15- Upset plot of gene sharing intersections.</p>
<p>16- Collage of genomic features of MPX virus focusing on new mutation. Modelled protein structure of mutant receptor atattched to viral membrane in complex to human receptor.</p>
<body>
  <div style="text-align:center;">
    <h1 style="color:green;">Education</h1>
  </div>
</body>
<hr style="width:100%;text-align:center;">
<br>
<br>
<p>Ongoing Doctorate degree in Bioinformatics (Federal University of Minas Gerais, Brazil).</p>
<p>Masters in Biochemistry and Molecular Biology (Federal University of Bahia, Brazil).</p>
<p>Bachelor in Biotechnology (Federal University of Bahia, Brazil with an exchange period at Carleton University, Canada).</p>
<br>
<body>
  <div style="text-align:center;">
    <h1 style="color:green;">Skills</h1>
  </div>
</body>
<hr style="width:100%;text-align:center;">
<br>
<p>
    <strong>Programming languages:</strong>
</p>
<h>Bash, Python, R (and a bit of HTML just to make this website <span>&#128517;</span>)</h>
<p>
    <strong>R packages:</strong>
</p>
<h>ggplot2, Pheatmap, Heatmaply, Reshape2, plotty, Upsetplot </h>
<p>
    <strong>Softwares:</strong>
</p>
<h>Genomics - FastQC, Figtree, MEGA, Aragorn, Prodigal, BLAST, GeneMarkS, MEME, CIAlign, SPAdes, IGV, Proksee, MAFFT, IQ-TREE, SAMtools, VEP </h> 
<br>
<h>Structural Biology - Pymol, Prodigy, Alphafold, Gromacs, Hdock, Firedock, QMEAN, OPM</h>
<br>
<h>Virology - VIRSCAN, VICTOR, PhaTYP, BACPHLIP, PHACTS, VipTree</h>
<br>
<h>Transcripomics - Proteinortho, miRWalk, hmmer, CD-HIT, Exatlas, Enrichr, Bowtie, GSEA, GO</h>
<br>
<h>Networks - Cytoscape, Gephi, STRING, CytoHubba, MCODE</h>
<p>
    <strong>Databases:</strong>
</p>
<h>GEO, PDB, NCBI, SRA, miRBase, UNIPROT, Alphafold, Virus-Host DB database, 1000 Genomes Project (1KG) </h>
<p>
    <strong>Other tools:</strong>
</p>
<h>Github, Collab, Inkscape</h>

<body>
  <div style="text-align:center;">
    <h1 style="color:green;">Publications</h1>
  </div>
</body>
<hr style="width:100%;text-align:center;">
<br>


<p><a href="https://www.tandfonline.com/doi/full/10.1080/07391102.2022.2042387" target="_blank">Human variation in the protein receptor ACE2 affects its binding affinity to SARS-CoV-2 in a variant-dependent manner</a></p>

<p><a href="https://www.sciencedirect.com/science/article/pii/S0888754319308535?via%3Dihub" target="_blank">The evolution of ADAM gene family in eukaryotes</a></p>

<p><a href="https://www.biorxiv.org/content/10.1101/2023.11.10.566534v1" target="_blank">Characterization and Phylogenetic Insights into the First Viral Major Vault Proteins (MVPs), Identified in Bacteriophages from the Human Gut</a></p>

<p><a href="https://journals.asm.org/doi/10.1128/jvi.01309-23" target="_blank">Gene duplication as a major force driving the genome expansion in some giant viruses</a></p>

<p><a href="http://dx.doi.org/10.51781/97865869201853250" target="_blank">Vigilância Genômica em Saúde Pública</a></p>

<p><a href="https://dasa.com.br/genov/" target="_blank">Relatório DASA - 2022</a></p>

<p><a href="https://www1.folha.uol.com.br/blogs/ciencia-fundamental/2022/04/diferencas-geneticas-afetam-como-o-coronavirus-se-liga-as-nossas-celulas.shtml?utm_source=sharenativo&utm_medium=social&utm_campaign=sharenativo" target="_blank">Diferencas genéticas afetam como o coronavírus se liga as nossas células</a></p>

<p><a href="https://www.em.com.br/app/noticia/bem-viver/2022/05/03/interna_bem_viver,1363929/covid-19-mutacao-em-proteina-humana-pode-alterar-a-afinidade-de-variantes.shtml" target="_blank">COVID-19: Mutação em proteina humana pode alterar a afinidade de variantes</a></p>
